.github/workflows/main.yml

```diff
@@ -178,7 +178,24 @@ jobs:
 
       - name: Run Clang-Tidy
         if: ${{ matrix.name == 'Clang-Tidy' }}
-        run: |
-          run-clang-tidy -j4 -quiet \
-            -checks="-*,clang-analyzer-*,-clang-analyzer-cplusplus*,-clang-analyzer-security*" \
-            -p build
+        uses: cpp-linter/cpp-linter-action@v2
+        id: cpp-linter
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          version: 18
+          step-summary: true
+          files-changed-only: false
+          style: ''  # disable clang-format checks.
+          tidy-checks: >-
+            -*
+            ,clang-analyzer-*
+            ,-clang-analyzer-cplusplus*
+            ,-clang-analyzer-security*
+            ,-clang-analyzer-valist*
```

Comment:
Iy seems `valist` checks are buggy: https://github.com/llvm/llvm-project/issues/40656

Reply:
So, are you going disable them?

Reply:
I already disabled them. `-clang-analyzer-*` means we enable all `analyzer` checks, `-clang-analizer-valist*` means we disable all `valist` checks.

Reply:
Ah, I overlooked the `-` prefix to the command.